### PR TITLE
Decouple JWT and LD signing/verifying paths

### PR DESCRIPTION
**Pull request by decentralgabe**
fixes https://github.com/TBD54566975/ssi-sdk/issues/194

Notably, this makes all endpoints use JWTs, leaving LD suites to be a bit lonely. LD suite support is *possible* but not integrated. Still thinking of the best way to include this...

### CODE REVIEW

**Comment by decentralgabe** [comment]
Added https://github.com/TBD54566975/ssi-sdk/issues/196 to address

File: crypto/jwt.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+func (sv *JWTSigner) GetSigningAlgorithm() string {
+	return sv.Algorithm()
+}
+
+// SignJWT takes a set of JWT keys and values to add to a JWT before singing them with the key defined in the signer
+func (sv *JWTSigner) SignJWT(kvs map[string]interface{}) ([]byte, error) {
+	t := jwt.New()
+	for k, v := range kvs {
+		if err := t.Set(k, v); err != nil {
+			err := errors.Wrapf(err, "could not set %sv to value: %v", k, v)
```
Comment (nitro-neal) [inline crypto/jwt.go:89]: %sv did you mean %v ?

Comment (decentralgabe) [inline crypto/jwt.go:89]:
```suggestion
			err := errors.Wrapf(err, "could not set %s to value: %v", k, v)
```

Comment (decentralgabe) [reply crypto/jwt.go]: %s, since the key is a string. fixed.

**Pull request merged**